src/ui/colorpicker.cpp

```diff
@@ -0,0 +1,81 @@
+#include "colorpicker.h"
+#include "ui_colorpicker.h"
+
+#include <QtWidgets>
+
+const int zoom_box_dimensions = 7;
```

Comment:
Unused warning for this, was there a plan for it?

Reply:
ah whoops, I was hard coding while trying to figure out the pixelRatio thing and forgot to switch back
resolved now